docs/adrs/adr-148-laravel-php-database-checks.md

```
@@ -1,6 +1,9 @@
+<!-- cpm:ignore-file STYLE-020 — documents the patterns these checks detect -->
 ---
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)